src/main/java/moe/yushi/authlibinjector/transform/support/YggdrasilKeyTransformUnit.java

```diff
@@ -139,6 +139,22 @@ public Optional<ClassVisitor> transform(ClassLoader classLoader, String classNam
 			return Optional.of(new ClassVisitor(ASM9, writer) {
 				@Override
 				public MethodVisitor visitMethod(int access, String name, String desc, String signature, String[] exceptions) {
+					if ("value".equals(name)) {
+						MethodVisitor mv = writer.visitMethod(access, "getValue", desc, signature, exceptions);
+						mv.visitVarInsn(ALOAD, 0);
+						mv.visitFieldInsn(GETFIELD, "com/mojang/authlib/properties/Property", "value", "Ljava/lang/String;");
+						mv.visitInsn(ARETURN);
+						mv.visitMaxs(1, 1);
```

Comment:
```suggestion
						mv.visitMaxs(-1, -1);
```
COMPUTE_MAXS enabled